tree.go

```diff
@@ -912,23 +896,6 @@ func (n *InternalNode) setDepth(d byte) {
 	n.depth = d
 }
 
-// MergeTrees takes a series of subtrees that got filled following
-// a command-and-conquer method, and merges them into a single tree.
-func MergeTrees(subroots []*InternalNode) VerkleNode {
-	root := New().(*InternalNode)
-	for _, subroot := range subroots {
-		for i := 0; i < NodeWidth; i++ {
-			if _, ok := subroot.children[i].(Empty); ok {
-				continue
-			}
-			root.TouchCoW(byte(i))
-			root.children[i] = subroot.children[i]
-		}
-	}
-
-	return root
-}
```

Comment:
I suspect this was used for some early approach of a full-tree conversion strategy.
Looks like a useful method, but if it doesn't seem to be useful today I think it might be better to remove it to avoid adding maintenance costs on code changes. We can always go back in git history and avoid redoing work if needed again.